entries/selectmenu.xml

```diff
@@ -73,7 +73,29 @@
 	"ui-selectmenu-button-open": "ui-corner-top",
 }</default>
 			<xi:include href="../includes/classes-option-desc.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
-			<xi:include href="../includes/classes-option-example.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
+			<examples>
+				<example>
+					<desc>Initialize the <placeholder name="name"/> with the <code>classes</code> option specified, changing the theming for the <code>ui-selectmenu-menu</code> class:</desc>
```

Comment:
Don't need the `<placeholder name="name"/>` here, could instead inline `selectmenu`. That's used a few more times below.